aws/sdk/examples/sns/src/bin/sns-hello-world.rs

```diff
@@ -3,16 +3,63 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-use sns::Region;
+use sns::{Client, Config, Region};
 use std::process::exit;
 
+use aws_types::region::ProvideRegion;
```

Comment:
More of a question for @rcoh than a blocker on this PR: should we be re-exporting ProvideRegion in each individual SDK so that explicit dependencies on aws-types aren't necessary?

Reply:
yeah we should have a quick design discussion about this (common?) use case of setting of a region chain. We can export as long as we aren't worried about it being an anti-pattern.